packages/@aws-cdk/aws-cloudwatch/lib/metric.ts

```diff
@@ -129,7 +129,7 @@ export interface MathExpressionOptions {
   /**
    * Label for this metric when added to a Graph in a Dashboard
    *
-   * @default - Expression value is used as label
+   * @default - The legend will only show the original metric labels
```

Comment:
If we change the default here, won't customers be surprised that their stacks have changed upon update? That seems problematic for a stable module. 

> To have the legend show only the original metric labels, set Expression-Label to be empty.

Instead of omitting the label entirely, can you set `label = ''`? To me, that seems to follow the cloudformation docs of explicitly setting the label to be empty.

Reply:
> If we change the default here, won't customers be surprised that their stacks have changed upon update? That seems problematic for a stable module.

I would argue that the current default behavior is unexpected, but I see your point  about the stability of the API and limiting the change to a specific input

Reply:
I like Neta's behavior better, actually, so I would recommend doing that. My apologies for implementing the original thing (shittily).

If we are concerned about introducing unexpected changes, I would recommend going with a feature flag instead.